schema/queue/configuration/FilterConfiguration.go

```diff
@@ -0,0 +1,76 @@
+package configuration
+
+import (
+	"fmt"
+	"log"
+)
+
+const (
+	FIELDNAME     string = "fieldName"
+	VALUE         string = "value"
+	OPERATION     string = "operation"
+	EXPECTEDVALUE string = "expectedValue"
+)
+
+type FilterFieldsConfiguration struct {
+	FieldName     string
+	ExpectedValue string
+	Operation     FilterOperation
+}
+type FilterOperation string
+
+const (
+	EQUAL     FilterOperation = "EQUAL"
+	NOT_EQUAL FilterOperation = "NOT_EQUAL"
+	EMPTY     FilterOperation = "EMPTY"
+	NOT_EMPTY FilterOperation = "NOT_EMPTY"
+	WILDCARD  FilterOperation = "WILDCARD"
+)
+
+func ConfigureFilters(filters []MqRouterFilterConfiguration) {
+	for _, filter := range filters {
+		metadatas := []FilterFieldsConfiguration{}
+		switch mType := filter.Metadata.(type) {
+		case map[string]interface{}:
```

Comment:
I think we must implement UnmarshalJSON (for json deserialization). Because casting `interface{}` to a type is not really good...